tests/cypress/support/commands.js

```diff
@@ -38,16 +38,29 @@ Cypress.Commands.add(
     var user = OPTIONS_HUB_USER || Cypress.env('OPTIONS_HUB_USER')
     var password = OPTIONS_HUB_PASSWORD || Cypress.env('OPTIONS_HUB_PASSWORD')
     var idp = OPTIONS_HUB_OC_IDP || Cypress.env('OPTIONS_HUB_OC_IDP')
-    cy.visit('/multicloud/home/search')
-    cy.get('body').then((body) => {
-      // Check if logged in
-      if (body.find('#header').length === 0) {
-        // Check if identity providers are configured
-        if (body.find('form').length === 0) cy.contains(idp).click()
-        cy.get('#inputUsername').click().focused().type(user)
-        cy.get('#inputPassword').click().focused().type(password)
-        cy.get('button[type="submit"]').click()
-        cy.get('.pf-c-page__header', { timeout: 30000 })
+
+    cy.visit('/multicloud/home/welcome')
```

Comment:
Could we make this URL and argument so each test can go directly to the page it needs without adding an extra step of loading welcome?
You could use it like:
```js
cy.visitAndLogin('/multicloud/home/search')
```

Reply:
or maybe call `cy.login()` from `searchPage.whenGoToSearchPage()`.  These 2 are called together in most tests, so combining it should make it easier and prevent the mistake of missing the call to `login()`

Reply:
I think combining them into a single function would work pretty well. I'll make the changes.